tests/integration/test_smoke.py

```diff
@@ -87,14 +86,16 @@
 # we've sent invalid params.
 ERROR_TESTS = {
     's3': {'ListObjects': {'Bucket': 'thisbucketdoesnotexistasdf'}},
+    'devicefarm': {'GetDevice': {'arn': 'arn:aws:devicefarm:REGION::device:f'}},
     'dynamodb': {'DescribeTable': {'TableName': 'unknowntablefoo'}},
     'sns': {'ConfirmSubscription': {'TopicArn': 'a', 'Token': 'b'}},
 }
 
 
-REGION_OVERRIDES = {
-    'devicefarm': 'us-west-2',
-}
+def _get_region(service, default='us-east-1'):
+    return {
+        'devicefarm': 'us-west-2',
```

Comment:
What's the motivation for removing `REGION_OVERRIDES` and recreating the override dictionary each time _get_region is called?  Why not keep `REGION_OVERRIDES` and just have `_get_region` be: `return REGION_OVERRIDES.get(service, default)`?